doc/Extensions/Authentication.md

```diff
@@ -182,17 +182,20 @@ An example config setup for use with Jumpcloud LDAP as a service is:
 
 ```php
 $config['auth_mechanism'] = "ldap";
+unset($config['auth_ldap_group']); #Remove built-in group
+unset($config['auth_ldap_groups']); #Remove built-in groups
```

Comment:
Instead of unset(), I'd say overwrite them like this:

```php
$config['auth_ldap_groups'] = [
    '{group}' = ['level' => 10];
];
```

Reply:
This code block works, but I only have one group for my LibreNMS admins.  My suggested change is for a basic example.  If you don't unset the variable it just appends your desired group to the defaults.  Perhaps include this codeblock in an advanced example?

Reply:
```php
$config['auth_ldap_groups'] = [
    '{admin_group}' => ['level' => 10],
];
```

boom, only one group.  I added the second to illustrate how multiple groups would be added.  (rather than duplicating the entire entry like the current docs)

Reply:
Honestly, it is probably best if we remove the default groups.  I can't imagine why they are set.